great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -1577,10 +1593,62 @@ def get_validator(
         include_rendered_content: Optional[bool] = None,
         **kwargs: Optional[dict],
     ) -> Validator:
-        """
-        This method applies only to the new (V3) Datasource schema.
-        """
+        """Retrieves a validator with a Batch list and an Expectation Suite.
 
+        `get_validator` first calls `get_batch_list` to retrieve a batch list, then creates or retrieves
+        an ExpectationSuite used to validate the Batches in the list.
+
+        Args:
+            datasource_name: The name of the Datasource that defines the Data Asset to retrieve the batch for
+            data_connector_name: The Data Connector within the datasource for the Data Asset
+            data_asset_name: The name of the Data Asset within the Data Connector
+            batch: The Batch to use with the Validator
+            batch_list: The List of Batches to use with the Validator
+            batch_request: Encapsulates all the parameters used here to retrieve a BatchList. Use either
+                `batch_request` or the other params (but not both)
+            batch_request_list: A List of BatchRequest to use with the Validator
+            batch_data: Provides runtime data for the batch; is added as the key `batch_data` to
+                the `runtime_parameters` dictionary of a BatchRequest
```

Comment:
Similar comment on the references to "BatchRequest". Either this is a class with backticks or we should put a space.